src/server/frontend_xwayland/xwayland_surface.cpp

```diff
@@ -137,7 +137,7 @@ auto property_handler(
     xcb_atom_t property,
     std::function<void(T const&)> handler) -> std::pair<xcb_atom_t, std::function<std::function<void()>()>>
 {
-    return property_handler<T>(connection, window, property, {handler});
+    return property_handler<T>(connection, window, property, mf::XCBConnection::Handler<T>{std::move(handler)});
```

Comment:
This doesn't seem to help with a segfault I'm getting on the performance tests on the Pi:
```
0x0000fffff7db3f04 in std::function<void (std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)>::function(std::function<void (std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)> const&) (
    this=0xffffd8ffb048, __x=...) at /usr/include/c++/9/bits/std_function.h:263
263         bool _M_empty() const { return !_M_manager; }
(gdb) bt
#0  0x0000fffff7db3f04 in std::function<void (std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)>::function(std::function<void (std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)> const&)
    (this=0xffffd8ffb048, __x=...) at /usr/include/c++/9/bits/std_function.h:263
#1  0x0000fffff7dbe8e0 in (anonymous namespace)::property_handler<std::__cxx11::basic_string<char> >(const std::shared_ptr<mir::frontend::XCBConnection> &, xcb_window_t, xcb_atom_t, std::function<void(const std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&)>) (connection=
    std::shared_ptr<mir::frontend::XCBConnection> (use count 4, weak count 0) = {...}, window=window@entry=4194306, property=property@entry=67, handler=...) at ./src/server/frontend_xwayland/xwayland_surface.cpp:134
#2  0x0000fffff7dbe8f8 in (anonymous namespace)::property_handler<std::__cxx11::basic_string<char> >(const std::shared_ptr<mir::frontend::XCBConnection> &, xcb_window_t, xcb_atom_t, std::function<void(const std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&)>)
    (connection=std::shared_ptr<mir::frontend::XCBConnection> (use count 4, weak count 0) = {...}, window=window@entry=4194306, property=property@entry=67, handler=...) at ./src/server/frontend_xwayland/xwayland_surface.cpp:134
# and so on
```